mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -2325,6 +2326,15 @@ class GroupGetLocalLinearRangePattern
 //===----------------------------------------------------------------------===//
 
 void mlir::populateSYCLToLLVMTypeConversion(LLVMTypeConverter &typeConverter) {
+  typeConverter.addTypeAttributeConversion(
+      [](BaseMemRefType, AccessAddrSpaceAttr addrSpace)
+          -> TypeConverter::AttributeConversionResult {
+        return IntegerAttr::get(
+            IntegerType::get(addrSpace.getContext(), 64),
```

Comment:
should we use index type here?

Reply:
I don't think that makes any difference here, as this only affects the width of the number encoding the memory space. We can even use 32 bits. WDYT?

Reply:
True, approved!